src/universal/modules/outcomeCard/containers/OutcomeCard/OutcomeCardContainer.js

```diff
@@ -36,6 +36,8 @@ class OutcomeCardContainer extends Component {
     super(props);
     const {outcome: {content}} = props;
     this.state = {
+      isEditingText: false,
+      isEditingMeta: false,
```

Comment:
Aha! Oops! You know what - I think that the "general" solution could dovetail really nicely into managing when the `GraphQL` mutation queries are fired off!